Source/WebKit/Platform/IPC/Encoder.cpp

```diff
@@ -72,17 +65,8 @@ Encoder::Encoder(MessageName messageName, uint64_t destinationID)
     encodeHeader();
 }
 
-Encoder::~Encoder()
-{
-    freeBufferIfNecessary();
-    // FIXME: We need to dispose of the attachments in cases of failure.
-}
-
-void Encoder::freeBufferIfNecessary()
-{
-    if (m_capacityBuffer.data() != m_inlineBuffer.data())
-        freeBuffer(m_capacityBuffer);
-}
+// FIXME: We need to dispose of the attachments in cases of failure.
```

Comment:
Good catch! I guess we need to create that test case too.

Reply:
I merely moved this comment.

Reply:
Nerve wracking!